reckless_github/src/repository.rs

```diff
@@ -35,6 +35,7 @@ impl Repository for Github {
     /// Where to store the index is an implementation
     /// details.
     async fn init(&self) -> Result<(), RecklessError> {
+        println!("{} was initialized", self.name);
```

Comment:
@vincenzopalazzo Why does this not work? As I understand we implement the `Repository` trait, this should mean this custom behavior should work when we call `init()` from the test in `lib.rs`.